optimum/neuron/modeling_decoder.py

```diff
@@ -246,7 +246,7 @@ def _save_pretrained(self, save_directory: Union[str, Path]):
 
         if src_compiled_path is None:
             # The compiled model has never been serialized: do it now
-            self.model._save_compiled_artifacts(dst_compiled_path)
+            self.model.save(dst_compiled_path)
```

Comment:
Do all these stable APIs also exist in 2.15.2? Otherwise do we need to bump the transformers-neuronx's version in the setup?

Reply:
No they are already there (introduced in 2.15.0). I had not seen them, but the doc had indeed been updated.